master/internal/proxy/proxy_intg_test.go

```diff
@@ -0,0 +1,101 @@
+package proxy
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	proxyAuth = func(c echo.Context) (done bool, err error) {
+		return true, nil
+	}
+	serviceIDs = []string{"a", "b", "c"}
+	u          = url.URL{} // "localhost:8081" TODO CAROLINA
+)
+
+func register(t *testing.T, prTCP bool, unauth bool) {
+	for _, id := range serviceIDs {
+		DefaultProxy.Register(id, &u, prTCP, unauth)
+		if DefaultProxy.GetService(id) == nil {
+			t.Logf("failed to find registered service %s", id)
+		}
+	}
+	if len(DefaultProxy.Summary()) != len(serviceIDs) {
+		t.Logf("failed to register all services")
+	}
+}
+
+func unregister(t *testing.T) {
+	for _, id := range serviceIDs {
+		if DefaultProxy.GetService(id) == nil {
+			t.Logf("failed to find registered service %s", id)
+		}
+		DefaultProxy.Unregister(id)
+		if DefaultProxy.GetService(id) != nil {
+			t.Logf("failed to unregister service %s", id)
+		}
+	}
+	if len(DefaultProxy.Summary()) != 0 {
+		t.Logf("failed to unregister all services.")
+	}
+}
+
+func DefaultProxyLifecycle(t *testing.T) {
```

Comment:
I thought tests needed to start with `Test...` to get run

Reply:
Runs fine without, but adding 'Test' to be consistent with all other intg-tests